src/main.js

```diff
@@ -117,7 +117,7 @@ const fetch_starred_repos = (options, pages) => {
  * @param {*} options 
  */
 const search = (options) => {
-  (options.verbose) ? console.log(chalk.bold.green(`🕵    INFO: Searching for "${options.findParam}" in "${options.user}"'s starred catalogue`)) : null;
+  (options.verbose) ? console.log(chalk.bold.green(`🕵    INFO: Searching for "${options.findParam}" ${options.organization ? `(inside org: ${options.organization})`: '' } in "${options.user}"'s starred catalogue`)) : null;
```

Comment:
```suggestion
  (options.verbose) ? console.log(chalk.bold.green(`🕵    INFO: Searching for "${options.findParam}" ${options.organization ? `(belonging to org: ${options.organization})`: '' } in "${options.user}"'s starred catalogue`)) : null;
```

Reply:
nit: this is a bit hard to read, what do you think if we add another variable up to deal with this condition?

Reply:
Yup I agree, What do you think about it now?